### PR TITLE
SIDE-2172: Add self-test-loop coverage for selected attribute for text input and radio buttons

### PR DESCRIPTION
**Pull request by cshields-oliveai**
Setting `selected` attribute on Select and Radio whisper types changes default value.

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/tests/index.ts [hunk 1]
```diff
@@ -272,12 +272,13 @@ export const createAndDeleteFile = (): Promise<boolean> =>
...
     const filePath = './test.txt';
     const writeMode = 0o755;
+
     network
       .encode('some text')
       .then((encodedValue) => {
         setTimeout(() => {
           filesystem
-            .writeFile(filePath, encodedValue, filesystem.WriteOperation.overwrite, writeMode)
+            .writeFile({path: filePath, data: encodedValue, writeOperation: filesystem.WriteOperation.overwrite, writeMode: writeMode})
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/index.ts:281]: Seems the interface for `writeFile` was updated.

File: ldk/javascript/examples/self-test-loop/src/tests/index.ts [hunk 2]
```diff
@@ -596,6 +597,38 @@ export const simpleFormWhisper = (): Promise<boolean> =>
...
+      onClose: () => {
+        console.debug('closed');
+      },
+      components: [
+        {
+          label: 'Select a color',
+          options: ['green', 'red', 'blue'],
+          onSelect: (selected) => {console.log(`${selected} has been selected!`)},
+          type: whisper.WhisperComponentType.Select,
+          selected: 2,
```
[the remark this answers is not in the document]
Comment (cshields-oliveai) [reply ldk/javascript/examples/self-test-loop/src/tests/index.ts:613]: only default selected index.

File: ldk/javascript/examples/self-test-loop/src/tests/index.ts [hunk 3]
```diff
@@ -376,7 +377,7 @@ export const listenFile = (): Promise<boolean> =>
           .encode('some text')
           .then((encodedValue) => {
             filesystem
-              .writeFile(filePath, encodedValue, filesystem.WriteOperation.append, writeMode)
+              .writeFile({path: filePath, data: encodedValue, writeOperation: filesystem.WriteOperation.overwrite, writeMode: writeMode})
```
[the remark this answers is not in the document]
Comment (cshields-oliveai) [reply ldk/javascript/examples/self-test-loop/src/tests/index.ts:380]: Sorry!

**Comment by richseviora** [comment]
This PR title/description reads like we're doing some work but I think I only see tests being updated?

**Review by BrettMugglinOlive — APPROVED** [review]

**Review by RusGraf — APPROVED** [review]

**Pull request merged**